test/system/150-login.bats

```diff
@@ -241,6 +241,36 @@ function _test_skopeo_credential_sharing() {
     rm -f $authfile
 }
 
+@test "podman login -secret test" {
+      secret=$(random_string 10)
+      echo -n ${PODMAN_LOGIN_PASS} > $PODMAN_TMPDIR/secret.file
+      run_podman secret create $secret $PODMAN_TMPDIR/secret.file
+
+      run_podman login --tls-verify=false \
+               --username ${PODMAN_LOGIN_USER} \
+               --secret ${output} \
+               localhost:${PODMAN_LOGIN_REGISTRY_PORT}
+    is "$output" "Login Succeeded!" "output from podman login"
+    # Now log out
+    run_podman logout localhost:${PODMAN_LOGIN_REGISTRY_PORT}
+    is "$output" "Removed login credentials for localhost:${PODMAN_LOGIN_REGISTRY_PORT}" \
+       "output from podman logout"
+    run_podman secret rm $secret
+
+    # test using secred id as --username
+    run_podman secret create ${PODMAN_LOGIN_USER} $PODMAN_TMPDIR/secret.file
+    run_podman login --tls-verify=false \
+               --secret ${PODMAN_LOGIN_USER} \
+               localhost:${PODMAN_LOGIN_REGISTRY_PORT}
+    is "$output" "Login Succeeded!" "output from podman login"
+    # Now log out
+    run_podman logout localhost:${PODMAN_LOGIN_REGISTRY_PORT}
+    is "$output" "Removed login credentials for localhost:${PODMAN_LOGIN_REGISTRY_PORT}" \
+       "output from podman logout"
+
+    run_podman secret rm ${PODMAN_LOGIN_USER}
+}
+
 # END   cooperation with skopeo
```

Comment:
It would be good to have a test that established a secret, and then a subsequent login using a bad secret fails.

Reply:
Nice idea.  One positive and one negative test :+1: